src/main/java/database/DataBaseManager.java

```diff
@@ -15,7 +15,7 @@ public class DataBaseManager {
     private static final Logger LOGGER = LogManager.getLogger(DataBaseManager.class.getName());
 
     @PostConstruct
-    public void DataBaseManager() {
```

Comment:
This class overall should manage the creation of the tables, as it does
and should also manage the connection, one connection is created here and used by the other components that require to insert or select on the database